### DISCUSSION TITLE
Plugin Installation Error in IntelliJ (again)

### DISCUSSION THREAD

**Discussion by chylex** [community/community#46865] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi, it seems #17250 is happening again. Every time I try to update the plugin from `1.2.0.2322` (apparently released January 2023, so many months after #17250) it does not shutdown the agent and the update fails.

I know how to work around it, but I don't know what's causing it and there are no errors in IntelliJ logs related to copilot, so I figured I'd post to let you know / ask around if anyone else has the same issue.

![obrazek](https://user-images.githubusercontent.com/3685160/217842818-d9b14d6c-39ba-4989-bdc4-232aef7e8019.png)

**Comment by iamcjwilliams** [comment]
@chylex Visit [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue

**Reply by chylex** [reply]
I already did, I'm posting here because I don't know if it's *my* issue or a regression in the plugin.

> I know how to work around it, but I don't know what's causing it and there are no errors in IntelliJ logs related to copilot, so I figured I'd post to let you know / ask around if anyone else has the same issue.

**Comment by iamcjwilliams** [comment]
@chylex Thank you, our copilot team has been informed
